include/cantera/kinetics/ReactionRate.h

```diff
@@ -145,6 +155,12 @@ class ReactionRate : public ReactionRateBase
         update(data);
     }
 
+    virtual void updateTTe(double T, double Te) override {
+        throw CanteraError("ReactionRate::updateTe",
+                           "Not implemented by derived ReactionRate object.",
+                           "This is only used for ETempRate");
+    }
```

Comment:
This should be handled exclusively by `ETempData::update` (would `PlasmaData` apply for this??)